pkg/systemd/services/service-details.jsx

```diff
@@ -319,6 +332,45 @@ export class ServiceDetails extends React.Component {
                 });
     }
 
+    deleteAction() {
+        // Stop timer so we don't get race conditions when the unit is gone.
+        if (this.interval)
+            clearInterval(this.interval);
+
+        const promises = [];
+        if (this.props.unit.UnitFileState === "enabled") {
```

Comment:
IMHO it should be possible to delete disabled custom timers as wel. In fact, that seems like the more plausible use case even -- first disable it, then see if you are happy with this, and then either re-enable or delete.

I.e. please split this into disable and a separate runtime (stop/reset) block.